File: Inventory-app/android/app/src/main/java/com/wekanmdb/storeinventory/model/store/Stores.kt
package com.wekanmdb.storeinventory.model.store

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.Required
import org.bson.types.ObjectId

open class Stores(
    @PrimaryKey
    var _id: ObjectId=ObjectId(),
    var name: String = "",
    var address: String = "",
    var _partition: String = ""
) : RealmObject()

File: Inventory-app/ios/AppShowcase/MVVM/Models/Stores.swift
//
//  Stores.swift
//  AppShowcase
//
//  Created by Gagandeep on 06/09/21.
//

import Foundation
import RealmSwift

class Stores: Object {
    @Persisted(primaryKey: true) var _id: ObjectId
    @Persisted var _partition: String = ""
    @Persisted var name: String = ""
    @Persisted var address: String = ""
    
    convenience init(_id: ObjectId, name: String, address: String, _partition: String) {
        self.init()
        self._id = _id
        self._partition = _partition
        self.name = name
        self.address = address
    }
}
